api/mail/email-verification.js

```diff
@@ -538,6 +585,50 @@ module.exports = function(mongoose) {
     transporter.sendMail(mailOptions, callback);
   };
 
+    /**
+   * Send an email to the Cboard support reporting a public board.
+   *
+   * @func sendConfirmationEmail
+   * @param {string} email - the user's email address.
+   * @param {function} callback - the callback to pass to Nodemailer's transporter
+   */
+     var sendReportEmail = function(data, callback) {
+       const {reportedBoardData: reportedBD, whistleblowerData: wbD, reportReason} = data;
+
+      if(!wbD.whistleblowerName){
+        wbD.whistleblowerName = 'Anonymus';
+        wbD.whistleblowerEmail = 'Anonymus';
+        wbD.whistleblowerLanguage = 'en';
```

Comment:
I think that if the front end grants the language of the language.lang state of the whistleblower's (not yet). Instead of the app.userData state. Overwriting it with 'En' would cause us to lose information. Maybe we could leave it as undefined